app/src/main/java/com/example/marcinwisniewski/intive_training_app/viewmodel/ListViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.marcinwisniewski.intive_training_app.viewmodel
+
+import android.arch.lifecycle.ViewModel
+
+
+class ListViewModel() : ViewModel() {
```

Comment:
If the class is empty you can remove curly brackets. 
I think it would be better to don’t merge commented code.